src/assignment/views/AssignmentOverview.tsx

```diff
@@ -644,6 +652,26 @@ const AssignmentOverview: FunctionComponent<AssignmentOverviewProps> = ({
 	};
 
 	const getLabelOptions = (labelType: Avo.Assignment.LabelType): CheckboxOption[] => {
+		if (labelType === 'SHARE_TYPE') {
+			return compact([
+				{
+					label: tText('assignment/views/assignment-overview___gedeeld-met-mij'),
+					id: AssignmentShareType.GEDEELD_MET_MIJ,
+					checked: [...(query.selectedShareTypeLabelIds || [])].includes(
+						AssignmentShareType.GEDEELD_MET_MIJ
+					),
+				},
+				{
+					// add GEDEELD MET ANDEREN to this option
+					label: tText('assignment/views/assignment-overview___mijn-opdrachten'),
+					id: AssignmentShareType.NIET_GEDEELD,
+					checked: [...(query.selectedShareTypeLabelIds || [])].includes(
+						AssignmentShareType.NIET_GEDEELD
+					),
+				},
+			]);
+		}
```

Comment:
the currently contains both CLASS and LABEL because those labels are stored in the same database field

So you better put the share type labels in a separate function since that is a separate database field and has nothing to do with CLASS/LEBEL assignment labels

```typescript
	getShareTypeLabels = (): CheckboxOption[] => {
		return compact([
			{
				label: tText('assignment/views/assignment-overview___gedeeld-met-mij'),
				id: AssignmentShareType.GEDEELD_MET_MIJ,
				checked: [...(query.selectedShareTypeLabelIds || [])].includes(
					AssignmentShareType.GEDEELD_MET_MIJ
				),
			},
			{
				// add GEDEELD MET ANDEREN to this option
				label: tText('assignment/views/assignment-overview___mijn-opdrachten'),
				id: AssignmentShareType.NIET_GEDEELD,
				checked: [...(query.selectedShareTypeLabelIds || [])].includes(
					AssignmentShareType.NIET_GEDEELD
				),
			},
		]);
	}
```